### ISSUE TITLE
TypeScriptでnew FirestoreSimpleの型がエラーになる

### ISSUE THREAD

**Issue by deutzia8152** [Kesin11/Firestore-simple#26]
firebase-adminの場合は正常に動きます。
```js
import admin from 'firebase-admin'
const firestore = admin.firestore()
const firestoreSimple = new FirestoreSimple(firestore)
```

下記は型エラーが発生します。
「'firebase.firestore.Firestore' の引数を型 'FirebaseFirestore.Firestore' のパラメーターに割り当てることはできません。
  Type 'Firestore' is missing the following properties from type 'Firestore': getAll, listCollections」
```js
import firebase from 'firebase/app'
import '@firebase/firestore'
const firestore = firebase.firestore()
const firestoreSimple = new FirestoreSimple(firestore)
```

**Comment by Kesin11** [comment]
I was able to reproduce it too. I found two of javascript SDK web and admin has different types.

web: https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore-types/index.d.ts
admin: https://github.com/googleapis/nodejs-firestore/blob/master/types/firestore.d.ts

Unfortunately, firestore-simple only support admin sdk types now.
I'll try to support web sdk while keeing the current interface as much as possible.
